### DISCUSSION TITLE
Understanding the --crop option

### DISCUSSION THREAD

**Discussion by emerylin** [quietvoid/dovi_tool#74] [Q&A]
My understanding is that --crop should be used if I am going to be injecting the RPU into an encoded video with the black bars cropped out. But why doesn't the --crop option also accept the specific cropping dimensions used in the encode (i.e. --crop 0:0:128:128)?

**Comment by quietvoid** [comment]
Because the dimensions aren't necessary.
`--crop` sets the active area offsets to 0, it doesn't need anything else.

Now if your video still has letterbox bars, you would have to edit the RPU accordingly, but only if the resolution changed.
